packages/react-dom/src/client/DOMPropertyOperations.js

```diff
@@ -172,3 +180,17 @@ export function setValueForProperty(
     }
   }
 }
+
+// Only used in DEV, stringifies a value and Warns if it took too long
+const stringifyWithPerformanceWarning = value => {
```

Comment:
For consistency let's move this to the top of the file, and then only define it in DEV

```js

let stringifyWithPerformanceWarning;

if (__DEV__) {
  stringifyWithPerformanceWarning = function(value) {
   // ...
  }
}
``